server/controllers/request.js

```diff
@@ -62,8 +64,11 @@ exports.createRequest = asyncHandler(async (req, res, next) => {
       "Something went wrong with your request please try again later"
     );
   }
-
-  res.status(200).send(request);
+  const requestSuccess = {
+    message: "Your request has been sent",
+    request,
+  };
+  res.status(200).send(requestSuccess);
```

Comment:
This introduces breaking changes same as stated above. Also I don't think adding a message is necessary, if a request is successfully created a request object is sent back to the client. you can either check for this or check the status of the server response (which will be `200` if no problems are encountered), you can then display the message you want.

Please Treat the request controller as a fire and forget, just send a `userId`  a start and end date, the original implementation will perform all necessary validation and created a request. otherwise it will throw an error. confirm a successful creation by checking the status or the returned object. 🙂

Reply:
@mayanksuchde, In that case, I'll update the booking UI